admin/css/picker.css

```diff
@@ -21,6 +21,25 @@
 
 .archive-picker {
   width: 100%;
+  display: flex;
+  height: 94vh;
+  gap: 10px;
+}
+
+.archive-picker-main {
+  position: relative;
+  flex: 100%;
```

Comment:
I assume you want to implement `flex-basis` here, right? If yes, I'd prefer to directly do `flex-basis: 30%` instead of `flex: 30%`.

Reply:
The `flex` is just shorthand that incorporates `flex-shrink`, `flex-grow` and `flex-basis`, so it makes the code cleaner in my opinion, what do you think?

Reply:
I agree that "yes" in some cases but some others "no". 
Just to clarify my point, if you want to apply all that properties then it's OK to use a shorthand and then use `flex`, however, if your decision is to apply ONLY `flex-basis` or `flex-grow` then I'd prefer to define separately.